html/elements/iframe.json

```diff
@@ -521,10 +521,10 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "76"
+                "version_added": "77"
```

Comment:
77 as opposed to 76 confirmed by https://storage.googleapis.com/chromium-find-releases-static/0fc.html#0fc50ca2d3c81f74f73aae8aab6079a516d9b8fb

76 came from https://github.com/mdn/browser-compat-data/pull/5442 and also did the same for `img`, so there's more data to fix. I've commented in https://github.com/mdn/browser-compat-data/pull/5442#issuecomment-742852922